### ISSUE TITLE
Deactivate NFD management in gpu-operator

### ISSUE THREAD

**Issue by geoberle** [NVIDIA/gpu-operator#87]
We have the NFD already installed in our cluster in another namespace. When we install the GPU operator in a dedicated namespace, it creates it's own subscription to NFD. The result is the NFD running twice per cluster. 

Is there a way to prevent his from happending?

**Comment by shivamerla** [comment]
@geoberle  Please set nfd.enabled=false in values.yaml used for Helm deployment. It's enabled by default.

**Comment by geoberle** [comment]
Thanks for you answer. I forgott to mention that i install via the Operator Lifecycle Manager, so no Helm involved. Is there a way of doing it as well?

**Comment by shivamerla** [comment]
@geoberle Its still in the works to control this through ClusterPolicy CR. In the CSV bundle we have a dependency on NFD CRD API version v1alpha1. But OLM will install it if this is missing, not sure why its deploying if its present already. Will look into this.

```
    required:
    - description: Discovers special resources and labels the nodes with the PCI vendor
        ID.
      displayName: Node Feature Discovery
      kind: NodeFeatureDiscovery
      name: nodefeaturediscoveries.nfd.openshift.io
      version: v1alpha1
```

**Comment by shivamerla** [comment]
@geoberle this is addressed in new operator v1.3.x published in RH catalog. NFD operator is no longer automatically deployed  when GPU operator is deployed through catalog. NFD has be installed prior to GPU operator.

**Comment by geoberle** [comment]
Thank you for the effort and the update 👍👌
